frontend/public/components/modals/tags.tsx

```diff
@@ -0,0 +1,73 @@
+import * as _ from 'lodash-es';
```

Comment:
Refactored to TypeScript and functional component using `withHandlePromise` HOC.